app/services/katello/pulp3/repository/yum.rb

```diff
@@ -368,12 +381,14 @@ def modular_packages(source_repository, filters)
           list_ids
         end
 
-        def additional_content_hrefs(source_repository, content_unit_hrefs)
+        def additional_content_hrefs(source_repository, content_unit_hrefs, copy_errata = false)
```

Comment:
I don't want the errata copied here since I copy it later: https://github.com/Katello/katello/pull/9556/files#diff-893a1c298832d5213af1a68f9aaae82f2507bd2414371bce47fa7c962d477089R312